tests/tracer/test_ext.py

```diff
@@ -23,5 +26,118 @@ def _updateenv(monkeypatch, env):
 
 @pytest.mark.parametrize("name,environment,tags", _ci_fixtures())
```

Comment:
Do the fixture files need to be updated to include the new environment variables/tags we are extracting from providers?

Reply:
I've reached out to the CI team, looks like we'll (or the CI team) have to update the fixture files.

Reply:
👍🏻 otherwise we should add unit tests for the new provider envs/tags we are adding.

Reply:
Haha the CI team will be making a PR for adding the new tags into the spec test files, will just need to update once it's merged 😄